### PR TITLE
Feature/userview modal

### LINKED ISSUE #283: App: Create UserView modal
We need a component that renders the User's profile. Follow the design on figma and use the right components (usually material-ui/react-native-paper)

### PR DESCRIPTION
**Pull request by joryclements**
Added feature to bring up user profile upon clicking their icon
https://expo.io/@joryclements/umigrate-userviewmodal
Closes #283

### CODE REVIEW

File: app/src/src/components/views/EventView.jsx [hunk 1]
```diff
@@ -35,9 +36,12 @@ const EventView = ({ item, updateItem }) => {
     interested,
   } = item;
 
+  const userView = useContext(UserViewContext);
   return (
     <Card style={sharedItemViewStyles.container}>
       <Card.Content style={sharedItemViewStyles.cardContent}>
+      <TouchableOpacity style={styles.row}
+        onPress = { () => userView.setUser(creator) } >
```
Comment (trentstauff) [inline app/src/src/components/views/EventView.jsx:44]: remove starting and trailing spaces ie { () => ... (creator) } (the ones beside the curly braces) for consistency

**Review by trentstauff — DISMISSED** [review]
Overall, great PR and awesome work for a nice first actual ticket!

Looks like the formatting checks are failing.

Run `npm run format:unix` in the src directory to fix this (https://github.com/Team-uMigrate/umigrate/wiki/uMigrate-App).

File: app/src/src/contexts/UserViewContext.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { useState, createContext } from 'react';
+
+const UserViewContext = createContext();
+
+// A context provider that stores the user whose profile is to be shown
```
Comment (deandrebaker) [inline app/src/src/contexts/UserViewContext.jsx:5]: Follow the same commenting structure and coding practices as the other Contexts.

**Review by deandrebaker — CHANGES_REQUESTED** [review]
@joryclements looking good so far! I added some requested changes below.

In addition, you forgot to update the AdView component with the TouchableOpacity that you added to the PostView.

**Review by deandrebaker — APPROVED** [review]

**Review by Apsd1109 — APPROVED** [review]
was fully reviewed by Deandre and Jory, ready to pull according to both.

**Pull request merged**